script/bootstrap.sh

```diff
@@ -17,19 +17,23 @@ setup_rbenv() {
   export PATH="$HOME/.rbenv/bin:$PATH"
   eval "$(rbenv init -)"
 
-  if [ ! -d "$HOME/.rbenv/versions/2.3.1" ]; then
-    subtask_exec "Install ruby 2.3.1" rbenv install 2.3.1
+  brew install rbenv
```

Comment:
Shouldn't this go above line 18?

My thought is that you could add a `brew bundle` step (between installing homebrew and setting up rbenv) and then add a `Brewfile` which specifies `rbenv`.

Or just try uninstalling rbenv and then running this script to make sure it all works 😉